### DISCUSSION TITLE
Congrats on v1.1

### DISCUSSION THREAD

**Discussion by shawnyeager** [otsaloma/catapult#33] [General]
I've been looking for a replacement for ulauncher on GNOME, and found your project [here](https://alternativeto.net/software/catapult-launcher/about/). I installed the Arch package, set a shortcut, and I'm good to go. Fast, clean, and well-designed. I look forward to seeing how it evolves. Cheers!

**Comment by otsaloma** [comment]
Nice to hear, thanks!

**Comment by shawnyeager** [comment]
Quick question: is there a `toggle` switch yet, or is `esc` the only way to exit currently?

**Reply by otsaloma** [reply]
Esc is the only way currently. I think before GTK-4, we would also auto-close upon losing focus, but not sure, some window-related things were lost when going from GTK-3 to 4.
